Integrations/integration-SplunkPy.yml

```diff
@@ -215,14 +215,19 @@ script:
 
         reader = results.ResultsReader(oneshotsearch_results)
         res = []
+        dbot_scores = []
         for item in reader:
             if isinstance(item, results.Message):
                 demisto.results({"Type": 1, "ContentsFormat": "json", "Contents": json.dumps(item.message)})
             elif isinstance(item, dict):
+                if demisto.get(item, 'host'):
+                    dbot_scores.append({'Indicator': item['host'], 'Type': 'hostname', 'Vendor': 'Splunk', 'Score': 0, 'isTypedIndicator': True})
```

Comment:
only 0 `Score`?

Reply:
 yes, from splunk hostname is not necessary malicious